examples/mlee.py

```diff
@@ -0,0 +1,610 @@
+
+_VERSION = "1.0.0"
+
+"""
+Step 3: Change the class name to correspond to your <Your_Dataset_Name> 
+ex: "ChemProtDataset".
+
+Then, fill all relevant information to `BuilderConfig` which populates information about the class. You may have multiple builder configs (ex: a large dataset separated into multiple partitions) if you populate for different dataset names + descriptions. The following is setup for just 1 dataset, but can be adjusted.
+
+NOTE - train/test/dev splits can be handled in `_split_generators`.
+"""
+
+
+class MLEE(datasets.GeneratorBasedBuilder):
+    """Write a short docstring documenting what this dataset is"""
+
+    VERSION = datasets.Version(_VERSION)
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name=_SOURCE_VIEW_NAME,
+            version=VERSION,
+            description=_DESCRIPTION,
+        ),
+        datasets.BuilderConfig(
+            name=_UNIFIED_VIEW_NAME,
+            version=VERSION,
+            description="BigScience biomedical hackathon schema",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = _SOURCE_VIEW_NAME
+
+    _ENTITY_TYPES = {
+        "Anatomical_system",
+        "Cell",
+        "Cellular_component",
+        "DNA_domain_or_region",
+        "Developing_anatomical_structure",
+        "Drug_or_compound",
+        "Gene_or_gene_product",
+        "Immaterial_anatomical_entity",
+        "Multi-tissue_structure",
+        "Organ",
+        "Organism",
+        "Organism_subdivision",
+        "Organism_substance",
+        "Pathological_formation",
+        "Protein_domain_or_region",
+        "Tissue",
+    }
+
+    """
+    Step 4: Populate "information" about the dataset that creates a skeletal structure for an example within the dataset looks like.
+
+    The following data structures are useful:
+
+    datasets.Features - An instance that defines all descriptors within a feature in an arbitrary nested manner; the "feature" class must strictly adhere to this format. 
+
+    datasets.Value - the type of the data structure (ex: useful for text, PMIDs)
+
+    datasets.Sequence - for information that must be in a continuous sequence (ex: spans in the text, offsets)
+
+    An example is as follows for an ENTITY + RELATION dataset.
+
+    Your format may differ depending on what the dataset is. Please try to keep the extraction as close to the original dataset as possible. If you're having trouble adapting your dataset, please contact the community channels and an organizer will reach out!
+    """
+
+    def _info(self):
+
+        if self.config.name == _SOURCE_VIEW_NAME:
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "text_bound_annotations": datasets.Sequence(  # T line in brat, e.g. type or event trigger
```

Comment:
I think we are using list of dict as opposed to sequence of dict now (same comment for other element below)

Reply:
Changed for all occurrences in bb2c660abe44819de088b5ec375511f00838ee4e